utils/exec.js

```diff
@@ -0,0 +1,12 @@
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+
+async function execAsync (cmd) {
```

Comment:
I'm using spawn a process because, I'm deploying this proxy using the hardhat task which needs the hardhat context to execute the task. So, I can not call the task directly as it requires the hardhat configuration.
I agree that the private key should not be set as an argument, I'll update to use the key directly from the config.json.

Reply:
I updated the cli to deploy the proxies, now we can deploy them using the cli or the hardhat task directly.